package.xml

```diff
@@ -18,9 +18,9 @@
   <buildtool_depend>ament_cmake_python</buildtool_depend>
   <buildtool_depend>python_cmake_module</buildtool_depend>
 
-  <build_depend>builtin_interfaces</build_depend>
-  <build_depend>rclcpp</build_depend>
-  <build_depend>rclpy</build_depend>
+  <depend>builtin_interfaces</depend>
```

Comment:
As far as I can tell, `builtin_interfaces` is only ever used by the Python code in here.  Thus, I think it should be marked as an `exec_depend`, not a `depend`.

Reply:
You're absolutely right, which means the CMakeLists.txt is nonsense as well.